MREUnityRuntime/MREUnityRuntimeLib/Messaging/Payloads/Converters/UnsignedConverter.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Text;
+using MixedRealityExtension.Util;
+using Newtonsoft.Json;
+
+namespace MixedRealityExtension.Messaging.Payloads.Converters
+{
+    /// <summary>
+    /// Converts signed integers into bit-equivalent unsigned integers,
+    /// because JS doesn't have a concept of unsigned.
+    /// </summary>
+    public class UnsignedConverter : JsonConverter
```

Comment:
Are you sure this works with nullable types? I had to account for nullable Enum types in the DashFormattedEnumConverter. I think you might need to do the same here.

Reply:
You do need to account for nullable types.  The type check in CanConvert should use the same code that the DashFomrattedEnumConverter does.